### PR TITLE
Add Native ScVi Source

### PR DESCRIPTION
**Pull request by Vaedz7**
Changes
- Added SV to Generator Config
- Added SV programs in Native Programs
- Removed Surprise Trader (Marked as developer only)

Mass Purchase in Separate PR

### CODE REVIEW

**Comment by Mysticial** [comment]
Sorry for taking so long to get this. I guess to start off, let's remove all the build artifacts. Those don't normally get checked in with the repo - especially if the source is included as well.

File: GeneratorConfig/PokemonSwSh/GodEggDuplication.json [hunk 1]
```diff
@@ -1,25 +0,0 @@
-﻿{
```
Comment (Mysticial) [inline GeneratorConfig/PokemonSwSh/GodEggDuplication.json:1]: Don't delete this file. The program does actually exist, but I think it's hidden.

File: GeneratorConfig/PokemonSV/MassPurchase.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "0-ProgramName": "MassPurchase",
+    "1-Documentation": "Microcontroller/blob/master/Wiki/PokemonSV/MassPurchase.md",
+    "2-Description": "Purchase a Specified Amount of Items",
+    "3-Parameters": [
+        {
+            "00-Type": "BooleanCheckBox",
+            "01-Label": "<b>Pay with LP:</b>,
```
Comment (Mysticial) [inline GeneratorConfig/PokemonSV/MassPurchase.json:8]: Syntax

File: GeneratorConfig/PokemonSwSh/GodEggDuplication.json (hunk 1 above)
Comment (Vaedz7) [reply GeneratorConfig/PokemonSwSh/GodEggDuplication.json:1]: Added back

File: GeneratorConfig/PokemonSwSh/DaySkipperUS.json [hunk 3]
```diff
@@ -18,8 +18,8 @@
             "02-Declaration": "const uint16_t REAL_LIFE_YEAR",
             "03-MinValue": 2000,
             "04-MaxValue": 2060,
-            "98-Default": 2022,
```
[the remark this answers is not in the document]
Comment (Vaedz7) [reply GeneratorConfig/PokemonSwSh/DaySkipperUS.json:21]: Changed both US and EU years

File: GeneratorConfig/PokemonSV/MassPurchase.json (hunk 2 above)
Comment (Vaedz7) [reply GeneratorConfig/PokemonSV/MassPurchase.json]: Added end quote

**Comment by Mysticial** [comment]
You haven't removed all the build artifacts. The entire `obj/` is artifacts. Since you're still learning, whenever a change includes a ton of non-text files that were automatically created, they are usually build files.

**Comment by Vaedz7** [comment]
> You haven't removed all the build artifacts. The entire `obj/` is artifacts. Since you're still learning, whenever a change includes a ton of non-text files that were automatically created, they are usually build files.

Deleted obj folder. Removed build artifacts

**Pull request merged**